packages/amplication-client/src/Entity/EntityFieldForm.tsx

```diff
@@ -129,38 +131,45 @@ const EntityFieldForm = ({
             <DisplayNameField
               name="displayName"
               label="Display Name"
-              disabled={isDisabled}
+              disabled={isDisabled || isSystemData}
               required
             />
             <NameField name="name" disabled={isDisabled} required />
             <OptionalDescriptionField
               name="description"
               label="Description"
-              disabled={isDisabled}
+              disabled={isDisabled || isSystemData}
             />
-            <div>
-              <ToggleField
-                name="unique"
-                label="Unique Field"
-                disabled={isDisabled}
-              />
-            </div>
-            <div>
-              <ToggleField
-                name="required"
-                label="Required Field"
-                disabled={isDisabled}
-              />
-            </div>
-            <div>
-              <ToggleField
-                name="searchable"
-                label="Searchable"
-                disabled={isDisabled}
-              />
-            </div>
-            {!SYSTEM_DATA_TYPES.has(formik.values.dataType) && (
-              <DataTypeSelectField label="Data Type" disabled={isDisabled} />
+            {schema.title !== "Id" && (
```

Comment:
Why did we decide to hide all these fields when editing an ID? 
If I understand correctly, before this change, all these fields are visible but disabled, and now we decided to remove them completely.